### ISSUE TITLE
Problem on mobile view

### ISSUE THREAD

**Issue by Sebarkar** [BKWLD/vue-ssr-carousel#90]
Hi there. 
Got the problem with displaying on mobile view.
When slide sometimes I getting unpredictable error. 

`activeSlides:Array[0]
dragging:false
leftPeekingSlideIndex:undefined
rightPeekingSlideIndex:undefined
slideOrder:Array[10]
0:0
1:1
2:2
3:3
4:4
5:5
6:6
7:7
8:8
9:9
trackTranslateX:NaN
allSlidesCount:10
clonedSlides:Array[0]
hasClonedSlides:false
inactiveSlides:Array[10]
0:0
1:1
2:2
3:3
4:4
5:5
6:6
7:7
8:8
9:9
styles:undefined
uniqueSlidesCount:10`

`<ssr-carousel
      show-dots
      overflow-visible
      :responsive="responsive"
      gutter="0"
      peek-right="20"
    >
    <slide v-for="i in 10"/>
    </ssr-carousel>`

`<ssr-carousel
      show-dots
      overflow-visible
      :responsive="responsive"
      gutter="0"
      peek-right="20"
    >
    <slide v-for="i in 10"/>
    </ssr-carousel>`
    
    ` data() {
    return {
      responsive: [
        {
          maxWidth: 3000,
          slidesPerPage: 5,
        },
        {
          maxWidth: 1279,
          slidesPerPage: 4,
        },
        {
          maxWidth: 1024,
          slidesPerPage: 3,
        },
        {
          maxWidth: 767,
          slidesPerPage: 2,
        }
      ]
    }
  },`

**Comment by Sebarkar** [comment]
This bug in case when drag-and move outside screen.

**Comment by Sebarkar** [comment]
[69176b59-10f6-4f67-ad39-a512ec13803b.webm](https://user-images.githubusercontent.com/44950137/188864552-fa1e4550-dc34-4f66-b785-f2ec79fc6a18.webm)

**Comment by weotch** [comment]
I couldn't reproduce this issue in Chrome using it's touch or non-touch modes.  I also can't reproduce it on a hardware iPhone in Safari.  Can you still repro it?

https://user-images.githubusercontent.com/77567/192053920-e7bb8f12-18fc-44b4-b517-73c6570d72d1.mov

**Comment by Sebarkar** [comment]
Yeah the same. On my mobile devices also in Chrome. In any Iphone or Android emulations.
Its 100% hapends when you slowly draging to the end of left side screen and out.

[fa759287-81fe-4e84-8269-7b9f723afcb2.webm](https://user-images.githubusercontent.com/44950137/192063801-484e1c44-ce76-44f0-8f28-f95d2dac6206.webm)

**Comment by Sebarkar** [comment]
On dekstop I never seen this bug.
Just pick demension Any devices. Not responsive.
![2222](https://user-images.githubusercontent.com/44950137/192064226-1aec3e3c-6e4a-4c4f-8aa0-e9028b0a8068.png)

**Comment by weotch** [comment]
Man I still can't repro. This is from Chrome on my iPhone. 

https://user-images.githubusercontent.com/77567/192118674-0fbdc390-061c-4b61-a258-a669fbcb9509.MOV

**Comment by Sebarkar** [comment]
Just used clear Firefox on mobile, same...

https://user-images.githubusercontent.com/44950137/192127310-7555baa8-e239-4039-91cc-3af4827676df.mp4


https://user-images.githubusercontent.com/44950137/192127316-011e94f1-41fe-4fdb-8856-ea292bc5c448.mp4

**Comment by Sebarkar** [comment]
I tried to show all manipulation.
[22222hh.webm](https://user-images.githubusercontent.com/44950137/192248906-79913259-1224-413d-b807-64e9ffcd3a93.webm)

**Comment by javiBarrientos** [comment]
Hi, i don't know if what i gonna say can be helpful but when i see your last video @Sebarkar all the index got "NaN". I'm working with this package and get the same error.
